packages/docs/docs/getting-started/connect-mode.md

```diff
@@ -6,14 +6,14 @@ sidebar_position: 5
 
 **Connect Mode** allows you to use some (currently limited) features of the Radon IDE while developing with your own React Native / Expo dev server ([Metro](https://metrobundler.dev/)) on your own simulator / emulator, or physical device.
 
-In this mode, you don't use the [Radon Panel](/docs/getting-started/panel-mode), but instead Radon IDE connects automatically to
+In this mode, you don't use the [Radon Panel](/docs/getting-started/panel-mode), but instead Radon IDE connects automatically to your running Metro development server.
```

Comment:
```suggestion
In this mode, you don't use the [Radon Panel](/docs/getting-started/panel-mode), but instead Radon IDE connects automatically to your running applications.
```
maybe? This is supposed to be an introductory blurb, not a technical explanation.
It's fine as is.